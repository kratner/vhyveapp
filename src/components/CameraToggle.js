import React from "react";
import { MdVideocamOff, MdVideocam } from "react-icons/md";
import styled from "styled-components";

const CameraToggleContainer = styled.div`
  color: #fff;
  cursor: pointer;
  z-index: 10;
  &:hover {
    color: #fdde6b;
  }
`;

const CameraToggle = props => {
  return (
    <CameraToggleContainer className="cameratoggle">
      {//Check if message failed
      props.active ? (
        <MdVideocam
          onClick={props.onClick}
          size={props.size}
          title="Camera On"
        />
      ) : (
        <MdVideocamOff
          onClick={props.onClick}
          size={props.size}
          title="Camera Off"
        />
      )}
    </CameraToggleContainer>
  );
};

export default CameraToggle;
