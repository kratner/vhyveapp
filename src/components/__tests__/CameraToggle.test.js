import React from "react";
import { shallow } from "enzyme";
import CameraToggle from "../CameraToggle";

describe("CameraToggle", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<CameraToggle />);
    expect(wrapper).toMatchSnapshot();
  });
});
