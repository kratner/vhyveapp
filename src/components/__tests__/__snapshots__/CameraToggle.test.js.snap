// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CameraToggle renders correctly 1`] = `ShallowWrapper {}`;
